frontend-react/src/components/DataDashboard/FacilityProviderSubmitterDetails/FacilityProviderSubmitterTable.tsx

```diff
@@ -0,0 +1,103 @@
+import { transformDate } from "../../../utils/DateTimeUtils";
+
+import styles from "./FacilityProviderSubmitterTable.module.scss";
+
+const filterManagerDefaults: FilterManagerDefaults = {
+    sortDefaults: {
+        column: "collectionDate",
+        order: "DESC",
+    },
+};
+
+interface FacilityProviderSubmitterTableProps {
+    aggregatorTypeId: string;
+    aggregatorTypeName: string;
+}
+
+function FacilityProviderSubmitterTable(
+    props: FacilityProviderSubmitterTableProps
+) {
+    const featureEvent = `${FeatureName.REPORT_DETAILS} | ${EventName.TABLE_FILTER}`;
+    // const { aggregatorTypeId }: FacilityProviderSubmitterTableProps = props;
+    const data = [
+        {
+            reportId: "fd34d590-eb8f-412f-9562-0975f2c413e3",
+            total: "5",
+            batchReadyAt: "2022-09-28T22:21:33.801667",
+            expires: "2022-10-28T22:21:33.801667",
+        },
+        {
+            reportId: "fd34d590-eb8f-412f-9562-0975f2c413e3",
+            total: "1",
+            batchReadyAt: "2022-09-28T22:21:33.801667",
+            expires: "2022-11-28T22:21:33.801667",
+        },
+        {
+            reportId: "fd34d590-eb8f-412f-9562-0975f2c413e3",
+            total: "2",
+            batchReadyAt: "2022-09-28T22:21:33.801667",
+            expires: "2023-09-28T22:21:33.801667",
+        },
+    ];
+
+    const filterManager = useFilterManager(filterManagerDefaults);
+
+    const tableConfig: TableConfig = {
+        columns: [
+            {
+                dataAttr: "reportId",
+                columnHeader: "Report ID",
+            },
+            {
+                dataAttr: "batchReadyAt",
+                columnHeader: "Date sent to you",
+                transform: transformDate,
+            },
+            {
+                dataAttr: "expires",
+                columnHeader: "Available until",
+                transform: transformDate,
+            },
+            { dataAttr: "total", columnHeader: "Test results" },
+        ],
+        rows: data!!,
+    };
+
+    return (
+        <div className={styles.FacilityProviderSubmitterTable}>
+            <section id="facilities">
+                <h2>
+                    Your available reports including {props.aggregatorTypeName}
+                </h2>
+                <TableFilters
+                    startDateLabel="From: (mm/dd/yyy)"
+                    endDateLabel="To: (mm/dd/yyyy)"
+                    filterManager={filterManager}
+                    onFilterClick={({
+                        from,
+                        to,
+                    }: {
+                        from: string;
+                        to: string;
+                    }) =>
+                        trackAppInsightEvent(featureEvent, {
+                            tableFilter: { startRange: from, endRange: to },
+                        })
+                    }
+                />
+                <Table config={tableConfig} />
```

Comment:
I know this will cause some refactoring, but can we start using the new table in `/shared/Table/Table`? Happy to help reformat the data if need be.